nsdl-c/sn_nsdl_lib.h

```diff
@@ -583,7 +497,7 @@ extern int8_t set_NSP_address(struct nsdl_s *handle, uint8_t *NSP_address, uint1
  * \return  0   Success
  * \return  -1  Failed to indicate that internal address pointer is not allocated (call nsdl_init() first).
  */
-extern int8_t set_NSP_address_2(struct nsdl_s *handle, uint8_t *NSP_address, uint8_t address_length, uint16_t port, sn_nsdl_addr_type_e address_type);
+extern int8_t set_NSP_address(struct nsdl_s *handle, uint8_t *NSP_address, uint8_t address_length, uint16_t port, sn_nsdl_addr_type_e address_type);
```

Comment:
Good point, we added a IOTCLT-1322 task for this and the other cleanup items you suggested.